protocol/yield-optimizers/concentrators.md

```diff
@@ -1,3 +1,23 @@
 # Concentrators
 
-Concentrators re-direct earnings into different strategies creating structured yield pipelines.
+Fortress auto-concentrators (Concentrators) are ERC-4626 vaults that use certain
+farming strategies to provide the user with minimum exposure to vault assets. 
+
+Concentrators claim their strategy rewards periodically and swap their reward
+tokens into target assets, which are then invested into vaults that provide risk
+profiles different from the originating vault. That is how Concentrators
+minimize exposure to reward tokens, by not holding on to them, but repurposing
```

Comment:
```suggestion
minimize exposure to reward tokens, by not holding on to them, but repurposing
```

this whole section is problematic, maybe more like "That is how Concentrators leverage the investment attributes of the principal, while not leaving the entire position exposed to the risks of the initial investment"

Reply:
other than that looks good to me